src/engine/regexp.cpp

```diff
@@ -1276,7 +1276,7 @@ regprop( char *op )
     case CLOSE+7:
     case CLOSE+8:
     case CLOSE+9:
-        sprintf(buf+strlen(buf), "CLOSE%d", OP(op)-CLOSE);
+        snprintf(buf+strlen(buf), sizeof(buf), "CLOSE%d", OP(op)-CLOSE);
```

Comment:
Isn't this (and the one above) should be `sizeof(buf)-strlen(buf)`?

Reply:
You are correct. Fixed in latest commit.